packages/m2m-gateway/src/model/errors.ts

```diff
@@ -349,3 +350,13 @@ export function eserviceTemplateRiskAnalysisNotFound(
     title: "E-Service Template risk analysis not found",
   });
 }
+
+export function eserviceTemplateVersionNotInSuspendedState(
+  versionId: string
+): ApiError<ErrorCodes> {
+  return new ApiError({
+    detail: `E-Service Template Version ${versionId} is not in suspended state`,
+    code: "eserviceTemplateVersionNotInSuspendedState",
+    title: "E-Service Template Version not in suspended state",
+  });
+}
```

Comment:
Unneeded, see: https://github.com/pagopa/interop-be-monorepo/pull/2407#discussion_r2378242011

```suggestion
```